### DISCUSSION TITLE
Showing guidelines to users

### DISCUSSION THREAD

**Discussion by DuvelCorp** [seerr-team/seerr#576] [Ideas]
I'd like to show a tooltip with some guidelines for the users to request their stuff correctly. In particular to choose the right root folder. 

I've got 30 users, and libraries (root folders) for : Movies, Series French, Series English, Animation movie, Animation series, Documentaries, Plays (scene).
And most of the stuff that should go in very specific libraries always lands in either Movies or Series folders, and never elsewhere. And then I have to move this stuff manually from Radarr or Sonarr.
If I could just have the possibility to show 100 chars of text explaining which root folder to chose, somewhere in the request interface (and modifiable from the admin panel), I think it would solve all my issues :)

Thank you

**Comment by fallenbagel** [comment]
There is one way to achieve this. But not through jellyseerr. Outside of it. Through [wizarr](https://github.com/wizarrrr/wizarr). They have like a setup guide thing for your users and you can add custom pages to it too and stuff
